src/server/app.ts

```diff
@@ -1,12 +1,22 @@
 /* eslint-disable no-param-reassign */
 import { Context } from "koa";
+import { parse } from "url";
 
 import markup from "./markup";
 
-async function app(ctx: Context) {
+async function app(ctx: Context, next: () => Promise<void>) {
+  const urlObj = parse(ctx.url);
+
+  // skip 404s
+  if (urlObj.pathname && urlObj.pathname.match(/\.(jpe?g|png|gif|bmp|svg|ico|js|json|txt|css)$/i)) {
```

Comment:
I feel this is very fragile and seems like a quickfix more than a permanent solution 😕 

I agree with 404s being 404s though, just needs a more bulletproof solution